package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "retail-url-scraper",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
```

Comment:
I'm guessing we want to add DEBUG=? something, otherwise no console logs will be shown?

Reply:
What is the reason behind `./bin/www` file (looks like cause he is an executable?)
Our entry point can be something more obvious, like - `node ./app/start-server.js`